Docs Protocol merge (#71)
dexe-network/investment-contracts: docs/Protocol/Litepaper.md

- ***DistributionProposal***

This is the contract the governance can execute in order to distribute rewards proportionally among all the voters who participated in the certain proposal.

#

#### ContractsRegistry

The contract is responsible for the *CRUD* functionality of other contracts in the protocol.

#### CoreProperties

The purpose of this contract is to store system constant parameters.

#### Insurance

An insurance is needed to secure users' assets invested in pools. Any user can buy insurance for any amount in **DeXe** tokens and get ***10x*** insurance. An investor who has bought insurance can open an insured event and provide information indicating a loss of funds.

#### PriceFeed

Contract is used for finding the best path for token exchange on **DEXes**.

#### TraderPools

A trader can create his own pool (fund) by sending a transaction using the smart contract method. The pool can be public and private. In private pool, the trader enters the addresses in the whitelist that can invest in this fund, public is open for all investors.

There are **2** types of pool:

> - *Standart*: trade using whitelisted tokens

Reviewer: ```suggestion
- *Standard*: trade using whitelisted tokens
```

- *Invest*: invest into offchain assets 

Pool has its native **ERC20** token with unique symbols and can have several managers.

Maximum number of open positions - **25**.

Maximum number of investors in the pool - **1000**.

#### UserRegistry

*UserRegistry* contract is used to manage user`s profile.

## Protocol Fees

Of all commission received on trading, **DeXe** charges its own commission of ***30%*** and use it to immediately purchase the **DEXE** token.
Commission distribution:
- ⅓ to the treasury of the protocol
- ⅓ to the insurance fund

> - ⅓ to the token holders (in the form of dividends) 

Reviewer: ```suggestion
- ⅓ to the NFT holders (in the form of extra rewards for the activity) 
```

## Governance

**DeXe** protocol is governed by **DAO** community. Parameters of the protocol that can be modified through protocol governance:
- **DeXe** commission change
- Changing proportions of **DeXe** commission distribution

> - Adding a token to the whitelist

Reviewer: ```suggestion
- Adding a token to the whitelist/backlist
```

> - Upgrade contracts

Reviewer: ```suggestion
- Upgrading contracts
```

- Managing the limits on the number of users in pools
- Managing the maximum number of positions for traders

> - Changing the parameters of the trader's leverage formula

Reviewer: ```suggestion
- Changing the parameters of the trader's leverage formula
- Adjusting insurance parameters
```
